cli/src/commands/branch.rs

```diff
@@ -77,6 +80,10 @@ pub struct BranchListArgs {
     /// wouldn't have a local target.
     #[arg(long, short)]
     revisions: Vec<RevisionArg>,
+
+    /// Whether to list non-tracking remote branches
+    #[arg(long)]
+    include_untracked: bool,
```

Comment:
Also add short `-a`?
I can see myself using this not never, and a short (and mnemonically obvious) option would be great

And long could be `--all` too, but that's whatever

Reply:
`--all` can have different meaning (disable `-r REVSET` filter and/or show tracking remotes even if the target matches local), so I avoided using "all".

That said, we can implement `--all` with the above definition. It should be easier to type and understand the meaning than `--include-untracked`.

Reply:
I'd suggest `-U` for `--include-untracked`.

Reply:
Does that mean that you prefer `--include-untracked` over `--all`, or do you just mean that you suggest `-U` *if* we go with `--include-untracked`?

Reply:
The latter. By now, I updated my previous comment as well.